lib/embed/media_tag.rb

```diff
@@ -75,7 +75,7 @@ def media_wrapper(thumbnail: '', file: nil, &block)
              data-thumbnail-url="#{thumbnail}"
              data-duration="#{file.try(:duration)}">
           <div class='sul-embed-media-wrapper'>
-            #{access_restricted_overlay(file.try(:stanford_only?), file.try(:location_restricted?))}
+            #{access_restricted_overlay(file.try(:stanford_only?), file.try(:location_restricted?)) unless previewable_only}
```

Comment:
What if, instead of passing `preview_only` around, we only make the call to `access_restricted_overlay` above the `audio/video` markup in `media_element` (between lines 42 and 43 of this file)?